cloud/terraform/terraform_controller.py

```diff
@@ -0,0 +1,122 @@
+import os
+
+
+class TerraformController:
+    def __init__(self, cloud):
+        self.cloud = cloud
+
+        #TODO: where is this executed from? change path?
+        if os.path.exists("main.tf"):
```

Comment:
Call me picky, but I would make use of single quotes whenever possible, and leave double-quote usage whenever needed.

Reply:
I never know when to use which one, so thank. I'll stick with single quotes